src/transformers/models/gpt_bigcode/modeling_gpt_bigcode.py

```diff
@@ -212,10 +236,16 @@ def forward(
         encoder_attention_mask: Optional[torch.Tensor] = None,
         use_cache: Optional[bool] = False,
         output_attentions: Optional[bool] = False,
+        **kwargs,
     ) -> Union[
         Tuple[torch.Tensor, Optional[torch.Tensor]],
         Tuple[torch.Tensor, Optional[torch.Tensor], Tuple[torch.Tensor, ...]],
     ]:
+        if "padding_mask" in kwargs:
+            warnings.warn(
```

Comment:
Let's use `logger.warning_once` here so that the user can control the verbosity and only see's it on the first forward call